docs/FAQ.md

```diff
@@ -390,6 +389,10 @@ Codespaces web dashboard instead of connecting directly to it.
 On the other hand, code-server is free, open-source, and can be run on any
 machine with few limitations.
 
+## Does code-server have any security login validation?
+
+code-server only supports a single password and limits logins to two per minute plus twelve per hour. 
```

Comment:
```suggestion
code-server supports setting a single password limiting logins to two per minute and twelve per hour. 
```

a small tweak. we might want to have @khorne3 take a look through these docs too, if she has the bandwidth?

Reply:
Maybe `code-server supports setting a single password and limits logins to two per minute and twelve per hour` otherwise it sounds a bit like the password is the thing limiting logins